cluster/gce/util.sh

```diff
@@ -765,13 +765,23 @@ function construct-linux-kubelet-flags {
       flags+=" --kubeconfig=/var/lib/kubelet/bootstrap-kubeconfig"
       flags+=" --register-schedulable=false"
     fi
+    if [[ "${MASTER_OS_DISTRIBUTION}" == "ubuntu" ]]; then
+      # Configure the file path for host dns configuration
+      # as ubuntu uses systemd-resolved
+      flags+=" --resolv-conf=/run/systemd/resolve/resolv.conf"
```

Comment:
i think this was not needed on all versions of Ubuntu.
/cc @chrisohaver

Reply:
this CI job will not be supporting all the versions. We will just support 18.04 LTS and above. so we should be good.